src/js/redux/reducers/search/filters/programSourceFilterFunctions.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * programSourceFilterFunctions.js
+ * Created by Lizzie Salita 6/14/19
+ */
+
+/* eslint-disable import/prefer-default-export */
+// We only have one export but want to maintain consistency with other query modules
+export const updateSelectedSources = (state, sourceIdentifier) => {
```

Comment:
Same thing can we rename this first param to soemthing a bit more descriptive?

Reply:
Renamed to `currentSources`